internal/session/query.go

```diff
@@ -407,6 +407,25 @@ and
 and
 	session_state.start_time < wt_sub_seconds_from_now(@threshold_seconds)
 ;
+`
+	sessionCredentialRewrapQuery = `
+select distinct
+  cred.session_id,
+  cred.key_id,
+  cred.credential,
+  cred.credential_sha256
+from session
+  inner join session_credential cred
+    on cred.session_id = session.public_id
+      and cred.key_id = ?
+where session.project_id = ?
+`
+	sessionCredentialRewrapUpdate = `
+update session_credential
+	set credential = ?,
+		key_id = ?
+where session_id = ?
+	and credential_sha256 = ?;
```

Comment:
Can this be done with a gorm type update instead? This was the table that didn't have a primary key right?

Reply:
I will check to see if this can be done with a db.withwhere

Reply:
I don't think that's what I mean, If we end up adding primary keys, we shouldn't need any `WithWhere`. We should just try to make this table easier to use with our types in general.

Reply:
yeah, if we end up adding a pk to this table then it would do this automatically

Reply:
doesn't look like updates can be performed via gorm using `withWhere` without a primary key